powersimdata/data_access/launcher.py

```diff
@@ -195,18 +218,28 @@ def _launch(self, threads=None, solver=None, extract_data=True):
         :return: (*dict*) -- contains "output", "errors", "scenario_id", and "status"
             keys which map to stdout, stderr, and the respective scenario attributes
         """
-        sys.path.append(server_setup.ENGINE_DIR)
         from pyreisejl.utility import app
```

Comment:
We assume here that the engine is REISE.jl, which is fine for now. In the future, we might have other engines and we will have to access their own utility functions

Reply:
Can we move the import statement at the top of the file and just do `app.launch_simulation(...)`, `app.extract_scenario(...)` and `app.check_progress(...)` ?

Reply:
We could use `importlib.import_module` to assign it? Not sure if there is another way. I tried just moving the import statement to the constructor as-is, but that resulted in an import error.

Reply:
I was trying to avoid to have the same import statement in each function. If it is too complicated or what you propose generate more lines, let's move forward with the current implementation.

Reply:
Done. Usually prefer the standard imports since they can be detected by refactoring tools, etc, but this use case is not too bad, and already a bit "non standard".